src/index.rs

```diff
@@ -513,7 +544,7 @@ impl Index {
     ) -> Option<ConstId> {
         initializer_id
             .as_ref()
-            .and_then(|it| import_from.remove(it))
+            .and_then(|it| import_from.clone(it))
```

Comment:
was this a test? or is this really required for builtins?

Reply:
AFAIK it is required, the problem is that we reused IDs for different types, and if we delete one when importing it is lost to the others who used it.
Since we own the other index, it will be dropped/deleted anyway